kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -324,44 +337,65 @@ define([
                     jobArray.forEach((jobState) => updateJobStatusInTable(jobState));
                 },
             });
+
+            const batchId = jobManager.model.getItem('exec.jobState.job_id');
             const paramsRequired = [];
             const jobIdList = [];
             jobs.forEach((jobState) => {
-                jobIdList.push(jobState.job_id);
-                if (!jobManager.model.getItem(`exec.jobs.params.${jobState.job_id}`)) {
-                    paramsRequired.push(jobState.job_id);
+                if (!jobState.batch_job) {
```

Comment:
This code sets up the listeners for the job status table. To start with, we go through the stored params (if there are any) and identify any jobs where we don't yet have the param data.